storage/object/api-cli/bucket-policy.mdx

```diff
@@ -151,6 +151,15 @@ Bucket policies use a JSON-based access policy language and are composed of stri
                 "<BUCKET_NAME>",
                 "<BUCKET_NAME>/*"
             ]
+        },
+        {
+            "Sid": "Delegate PUT to application",
+            "Effect":"Allow",
+            "Principal":{
+                "SCW":"application_id:<APPLICATION_ID>"
+            },
+            "Action":"s3:PutObject",
+            "Resource":"<BUCKET_NAME>/*"
```

Comment:
Added a 2nd array to the "Statement" array in the Statement description to make syntax clearer